### PR TITLE
add resources to default imagepolicy list

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@openshift/team-documentation ptal

goes back as far as this topic has existed.

File: admin_guide/image_policy.adoc [hunk 1]
```diff
@@ -165,7 +165,7 @@ admissionConfig:
 <5> List of annotations to match on the image object's metadata.
 <6> If you are not able to resolve the image, do not fail the pod.
 <7> Array of rules allowing use of image streams in Kubernetes resources. The default configuration
-    allows pods, replication controllers, replica sets, deployments, and jobs
+    allows pods, replicationcontrollers, replicasets, statefulsets, daemonsets, deployments, and jobs
```
Comment (kalexand-rh) [inline admin_guide/image_policy.adoc]: It looks like both "replicationcontrollers" and "replication controllers" are used in the collection. Do we have a usage preference here?

Comment (bparees) [reply admin_guide/image_policy.adoc]: i'm not aware but the resource name has no space in it, so i went w/ that.

Comment (kalexand-rh) [reply admin_guide/image_policy.adoc]: That's a good reason. :) LGTM

**Comment by kalexand-rh** [comment]
I was mistaken - this line was added in 3.5, not 3.3.

**Pull request merged**